dev/openchpl_data-model.sql

```diff
@@ -588,11 +588,15 @@ CREATE TABLE openchpl.test_functionality (
 	number varchar(200) not null,
 	name varchar(1000),
 	certification_edition_id bigint NOT NULL,
+	practice_type_id bigint NULL,
```

Comment:
Should this be `practice_type_id bigint default null,`?

It might be valid, but looks odd. The way I usually test changes to these files is running the `reset-test-only.sh` script, and making sure no errors appear